src/experiment/experiment.js

```diff
@@ -119,348 +111,346 @@ export function buildExperiment() {
     ],
   };
 
-const jsPsych = initJsPsych({
-  on_finish: () => {
-    redirect(redirectTo);
-  },
-});
-
-
-
-const getPid = {
-  type: surveyText,
-  questions: [
-    {
-      prompt: 'Participant ID:',
-      name: 'pid',
-      placeholder: '0000',
-      required: true,
-    },
-    {
-      prompt: 'Class ID:',
-      name: 'ClassId',
-      placeholder: '0000',
-      required: true,
+  const jsPsych = initJsPsych({
+    on_finish: () => {
+      redirect(redirectTo);
     },
-    {
-      prompt: 'School ID',
-      name: 'SchoolId',
-      placeholder: '0000',
-      required: true,
+  });
+
+  const getPid = {
+    type: surveyText,
+    questions: [
+      {
+        prompt: 'Participant ID:',
+        name: 'pid',
+        placeholder: '0000',
+        required: true,
+      },
+      {
+        prompt: 'Class ID:',
+        name: 'ClassId',
+        placeholder: '0000',
+        required: true,
+      },
+      {
+        prompt: 'School ID',
+        name: 'SchoolId',
+        placeholder: '0000',
+        required: true,
+      },
+    ],
+    on_finish: (data) => {
+      participantId = [
+        data.response.SchoolId,
+        data.response.ClassId,
+        data.response.pid,
+      ].join('-');
+      classId = data.response.ClassId;
+      schoolId = data.response.SchoolId;
```

Comment:
Move out. I've been using getUserData.js for these trials, we can stick with that for now.